bucket/tumblthree.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "1.0.10.1008",
+    "homepage": "https://github.com/TumblThreeApp/TumblThree",
+    "description": "A Tumblr Backup Application.",
+    "license": "MIT",
+    "url": "https://github.com/TumblThreeApp/TumblThree/releases/download/v1.0.10.1008/TumblThree-v1.0.10.1008-Application.zip",
+    "hash": "0ddcf69b1944272a06212b7c3a529d8c177309f81ea266045142d1b36a357bac",
+    "extract_dir": "TumbleThree",
+    "pre_install": [
+        "foreach ($name in @('Cookies.json', 'Manager.json', 'Queuelist.json')) {",
+        "    if (!(Test-Path \"$persist_dir\\$name\")) { New-Item \"$dir\\$name\" -Force | Out-Null }",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\Settings.json\")) { Set-Content \"$dir\\Settings.json\" '{\"PortableMode\": true}' -Encoding ASCII -Force }"
```

Comment:
Why force? I think you forgot to add Settings into foreach. Because then you potentially need force.

Reply:
Should I remove both of the force?

Reply:
No need to use force in any of this code. Any file exist in archive.